[TEST-FAILURE] QueueStatisticsTest.testEmptyPollOperationCount

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-IbmJDK1.7/lastBuild/com.hazelcast$hazelcast/testReport/com.hazelcast.queue/QueueStatisticsTest/testEmptyPollOperationCount/

```
java.lang.AssertionError: expected:<10> but was:<9>
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.failNotEquals(Assert.java:743)
    at org.junit.Assert.assertEquals(Assert.java:118)
    at org.junit.Assert.assertEquals(Assert.java:555)
    at org.junit.Assert.assertEquals(Assert.java:542)
    at com.hazelcast.queue.QueueStatisticsTest.testEmptyPollOperationCount(QueueStatisticsTest.java:96)
```

--- a/hazelcast/src/test/java/com/hazelcast/queue/QueueStatisticsTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/queue/QueueStatisticsTest.java
@@ -4,6 +4,7 @@ import com.hazelcast.core.IQueue;
 import com.hazelcast.core.ItemEvent;
 import com.hazelcast.core.ItemListener;
 import com.hazelcast.monitor.LocalQueueStats;
+import com.hazelcast.test.AssertTask;
 import com.hazelcast.test.HazelcastParallelClassRunner;
 import com.hazelcast.test.annotation.QuickTest;
 import org.junit.Test;
@@ -63,6 +64,7 @@ public class QueueStatisticsTest extends AbstractQueueTest {
         assertEquals(10, stats.getRejectedOfferOperationCount());
     }
 
+
     @Test
     public void testPollOperationCount() throws InterruptedException {
         IQueue queue = newQueue();
@@ -80,8 +82,15 @@ public class QueueStatisticsTest extends AbstractQueueTest {
             queue.poll();
         }
 
-        LocalQueueStats stats = queue.getLocalQueueStats();
-        assertEquals(30, stats.getPollOperationCount());
+
+        final LocalQueueStats stats = queue.getLocalQueueStats();
+        AssertTask task = new AssertTask() {
+            @Override
+            public void run() throws Exception {
+                assertEquals(30, stats.getPollOperationCount());
+            }
+        };
+        assertTrueEventually(task);
     }
 
     @Test
@@ -92,8 +101,14 @@ public class QueueStatisticsTest extends AbstractQueueTest {
             queue.poll();
         }
 
-        LocalQueueStats stats = queue.getLocalQueueStats();
-        assertEquals(10, stats.getEmptyPollOperationCount());
+        final LocalQueueStats stats = queue.getLocalQueueStats();
+        AssertTask task = new AssertTask() {
+            @Override
+            public void run() throws Exception {
+                assertEquals(10, stats.getEmptyPollOperationCount());
+            }
+        };
+        assertTrueEventually(task);
     }
 
     @Test

